run/Flan/Modern Weapons Pack/vehicles/Abrams.txt

```diff
@@ -32,12 +32,30 @@ FuelTankSize 2000
 //Inventory Slots
 CargoSlots 0
 ShellSlots 5
+
```

Comment:
I'm not sure I support blank lines. It will probably just skip it, but not until it's asked every single data reader if they are looking for blank. Perhaps use a commented line instead?